style/layer/road.js

```diff
@@ -0,0 +1,367 @@
+//At this zoom, render switches from unified to differentiated bridge/tunnel rendering
+let minzoomBrunnel = 11;
+
+//Exponent base for inter-zoom interpolation
+let roadExp = 1.2;
+
+//Tunnel casing dash pattern
+let tunDashArray = [
+  "step",
+  ["zoom"],
+  ["literal", [1]],
+  minzoomBrunnel,
+  ["literal", [0.5, 0.25]],
+];
+
+//Join styles for fill and casing
+let layoutRoadFill = {
+  "line-cap": "round",
+  "line-join": "round",
+  visibility: "visible",
+};
+let layoutRoadCase = {
+  "line-cap": "butt",
+  "line-join": "round",
+  visibility: "visible",
+};
+
+//Highway class styles
+class Motorway {
+  constructor(link) {
```

Comment:
Looks like this `link` parameter is not used in practice.

Reply:
Removed useless constructor.